src/main/JayZ.ts

```diff
@@ -25,69 +31,99 @@ export class JayZ {
         ? config.encryptor
         : new LibsodiumEncryptor()
     this.maxUsesPerDataKey = config.maxUsesPerDataKey || 1
+    this.currentDataKeyUsesRemaining = this.maxUsesPerDataKey
   }
 
-  async encryptItem<T, K extends keyof T>(
-    item: T,
-    fieldsToEncrypt: K[]
-  ): Promise<EncryptedJayZItem<T, K>> {
-    const { dataKey, encryptedDataKey } = await this.getDataKey()
-    const { encryptedItem, nonce } = await this.encryptor.encrypt({
-      item,
-      fieldsToEncrypt,
-      dataKey
+  async encryptItems<T, K extends keyof T>(
```

Comment:
So now we have batch `encryptItems` and `decryptItems`

Reply:
Didn't see the package version bump yet, but I'm assuming we are planning to bump a major version since we're removing the singular methods (`encryptItem` and `decryptItem`).

Since Jay-Z is a public repo, do we publish a change log or otherwise communicate breaking changes?

Looks like the README references the singular methods, so we should update it.

Reply:
Good things to bring up: 

1. Re: versioning, semvar technically doesn't require major version bumps for breaking changes until the version is >= 1.0.0. So I think the question here is really more of a "should we just call this library 1.0x yet?". Imo I think this library will see a bit more iteration before we claim 1.x, but if you (or others) are more of the opinion of "just call this 1.x already damnit", I can be convinced :).
 

2. Re: README -- good catch. I had local changes for that, which I forgot to commit. Just added.